code/WorkInProgress/Cael_Aislinn/Rust/fuel_assembly_port.dm

```diff
@@ -4,7 +4,7 @@
 	name = "Fuel Assembly Port"
 	icon = 'code/WorkInProgress/Cael_Aislinn/Rust/rust.dmi'
 	icon_state = "port2"
-	density = 0
+	setDensity(FALSE)
```

Comment:
I *think* this is overriding the `setDensity `proc for this atom, instead of calling it. Thanks, BYOND.

Reply:
Relative patheting was a mistake
This shouldn't compile imo